### PR TITLE
Changes to make EditTenant frontend work

### PR DESCRIPTION
**Pull request by AdamWillford**

### CODE REVIEW

File: src/views/tenant/administration/Tenants.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ const columns = [
     name: 'Actions',
     center: true,
     cell: (row) => (
-      <Link to={`/tenant/administration/tenants/Edit?TenantFilter=${row.defaultDomainName}`}>
+      <Link to={`/tenant/administration/tenants/Edit?tenantDomain=${row.defaultDomainName}`}>
```
Comment (homotechsual) [inline src/views/tenant/administration/Tenants.js:116]: Why are we changing to `tenantDomain` when we use TenantFilter basically everywhere else?

**Review by homotechsual — CHANGES_REQUESTED** [review]

File: src/views/tenant/administration/Tenants.js (hunk 1 above)
Comment (KelvinTegelaar) [reply src/views/tenant/administration/Tenants.js:116]: Stick to tenantFilter plz,

File: src/views/tenant/administration/EditTenant.js [hunk 2]
```diff
@@ -32,17 +32,22 @@ const EditTenant = () => {
...
       setQueryError(true)
     }
   }, [tenantDomain, dispatch])
 
   const onSubmit = (values) => {
-    // @todo bind this
-    //window.alert(JSON.stringify(values))
-    genericPostRequest({ path: '/api/EditTenant', values })
+    const shippedValues = {
+      Tenant: tenantDomain,
```
Comment (KelvinTegelaar) [inline src/views/tenant/administration/EditTenant.js:44]: Your API is expecting tenantid: 

```powershell
$Tenant = $request.body.tenantid
```

File: src/views/tenant/administration/Tenants.js (hunk 1 above)
Comment (homotechsual) [reply src/views/tenant/administration/Tenants.js:116]: So the pattern is usually `TenantFilter` is the URL param and `tenantDomain` is the JS variable.

File: src/views/tenant/administration/EditTenant.js (hunk 2 above)
Comment (homotechsual) [reply src/views/tenant/administration/EditTenant.js:44]: This is why we have a consistent naming really :-) <3

**Pull request merged**